src/test/clojurescript/clara/test.cljs

```diff
@@ -12,4 +12,4 @@
     (println "FAIL")))
 
 (defn ^:export run []
-  (test/run-tests 'clara.test-rules 'clara.test-common))
+  (test/run-tests 'clara.test-rules 'clara.test-common 'clara.test-salience))
```

Comment:
I'm not too familiar with ClojureScript's compilation, but I think something analogous to Clojure's behavior where namespaces can be referenced with full qualification when they have already been compiled is happening here. I had to add the test-salience require to test.cljs for the build of this branch to pass on my local machine.  In any case it seems safest to explicitly require namespaces on which a namespace directly depends.